kempner_hpc_handbook/scalability/distributed_gpu_computing.md

```diff
@@ -0,0 +1,658 @@
+# Distributed GPU Computing
+The scaling of machine learning (ML) workloads in HPC environments can be achieved through various parallelism approaches. This section outlines the primary methods for parallelizing ML computations.
+
+One can decide to use multiple GPUs on their AI/ML applications for different reasons including but not limited to:
+* Handling large-scale datasets
+* Hyperparameter tuning
+* Train/inferece large-scale Model that does not fit into the memory of a single GPU. 
```

Comment:
Also we can add, 
- Achieving faster results, even if the data or model fits into a single GPU's memory.